credsweeper/deep_scanner/deep_scanner.py

```diff
@@ -1,7 +1,8 @@
+import datetime
 import logging
 from typing import List, Optional, Any, Tuple, Union
 
-from credsweeper.common.constants import RECURSIVE_SCAN_LIMITATION
+from credsweeper.common.constants import RECURSIVE_SCAN_LIMITATION, RuleType
```

Comment:
```suggestion
from credsweeper.common.constants import RECURSIVE_SCAN_LIMITATION
```
TODO: apply without CI before merge